src/main/java/org/fundacionjala/coding/dennis/FizzBuzz.java

```diff
@@ -0,0 +1,52 @@
+package org.fundacionjala.coding.dennis;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ *
+ */
+public  final class FizzBuzz {
+
+    public static final int THREE = 3;
+    public static final int FIVE = 5;
+    public static final int MCM = THREE * FIVE;
+
+    /**
+     *
+     */
+    private FizzBuzz() {
+    }
+
+    /**
+     *
+     * @param number numero a ser evaluado.
+     * @return cadena resultado.
+     */
+    public static String singleFizzBuzz(int number) {
+        if (number % MCM == 0) {
+            return "FizzBuzz";
+        } else if (number % THREE == 0) {
+            return "Fizz";
+        } else if (number % FIVE == 0) {
+            return "Buzz";
+        } else {
+            String fbuzz = Integer.toString(number);
+            CharSequence three = "3", five = "5";
+            return fbuzz.contains(three) ? "Fizz" : fbuzz.contains(five) ? "Buzz" : fbuzz;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Extract this nested ternary operation into an independent statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3358)